### ISSUE TITLE
Turtle tool minability checks are limited

### ISSUE THREAD

**Issue by kreezxil** [SquidDev-CC/CC-Tweaked#1464]
### Minecraft Version

1.19.x

### Version

1.101.2

### Details

Turtles can't mine a block that has no registered tool. altho the block can be easily mined by players with an empty hand or anything in their hand. It's my block **The Kabbalah Block** https://www.curseforge.com/minecraft/mc-mods/the-kabbalah-block

[latest.log](https://github.com/cc-tweaked/CC-Tweaked/files/11637902/latest.log)

**Comment by SquidDev** [comment]
Thank you for the report! What tool were you using to break the block? I had no problems using a mining turtle/diamond pick.

This is expected behaviour for non-mining turtles (such as the digging or chopping turtle), as they're more restrictive in what blocks they can dig.

**Comment by kreezxil** [comment]
yes, diamond mining turtle mines everything, it's the other pickaxes that don't work, my bad for not being clear on that. These pickaxes didn't work: wooden, stone, iron, netherite

**Comment by SquidDev** [comment]
Are you using a custom datapack with CC? Or possibly an add-on mod? We don't have non-diamond tools by default.

**Comment by kreezxil** [comment]
Yes, that's it, thanks for helping me to resolve this. I'll report it to them. 
![image](https://github.com/cc-tweaked/CC-Tweaked/assets/3880072/5e0df300-5546-4e66-bc87-286de582f531)

**Comment by OHakimen** [comment]
I got around to see the issue, because of the way the tiered tools work, i can't directly fix it without i either adding his block on the tier tags that i have (so stone picks don't break obsidian and etc...) or he could add a tag to this block to make it always harvestable (a.k.a adding their block to this tag computercraft:turtle_always_breakable)

**Comment by SquidDev** [comment]
See also #1562, where this occurred with Iron Pickaxes.

@JheffersonMarques I feel you're better off here writing your own turtle upgrade, rather than using CC's built-in `TurtleTool` here. It's not really flexible right now for what you want to do, and I'm not quite sure which way I want to go about solving it yet.

**Comment by OHakimen** [comment]
I think i'm gonna actually remove other tools, since it depends on tags (on current impl) there is always gonna be one or two blocks missing
